metricbeat/module/system/cpu/cpu.go

```diff
@@ -100,6 +100,11 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		"5":  loadStat.Load5,
 		"15": loadStat.Load15,
 	}
+	cpuStat["load_normalized"] = common.MapStr{
```

Comment:
I agree that they should be under `load`. I suggest `load.avg.1` and `load.normalized.1`.